examples/tutorials/advanced/focal_mechanisms.py

```diff
@@ -0,0 +1,336 @@
+    component="deviatoric",  # deviatoric part
+)
+
+fig.show()
+
+
+# %%
+# Adjusting the outlines
+# ----------------------
+#
+# Use the parameters ``pen`` and ``outline`` for adjusting the circumference of the
+# beachball or all lines (circumference of the beachball and both nodal planes).
+
+fig = pygmt.Figure()
+fig.basemap(region=region, projection=projection, frame=frame)
+
+fig.meca(
+    spec=aki_dict_single,
+    scale="1c",
+    longitude=-2,
+    latitude=0,
+    # Use a 1-point thick, darkorange and solid line
+    pen="1p,darkorange,solid",
+)
+
+fig.meca(
+    spec=aki_dict_single,
+    scale="1c",
+    longitude=2,
+    latitude=0,
+    outline="1p,darkorange,solid",
+)
+
+fig.show()
+
+
+# %%
+# Highlighting the nodal planes
+# -----------------------------
+#
+# Use the parameter ``nodal``, whereby ``"0"`` refers to both, ``"1"`` to the first,
+# and ``"2"`` to the second nodal plane(s). Only the circumference and the specified
+# nodal plane(s) are plotted, i.e. the quadrants remain unfilled (transparent). If
+# needed, make usage of the stacking concept of (Py)GMT and use ``nodal`` with the
+# ``outline`` or / and ``pen``  parameters in combination.
+
+fig = pygmt.Figure()
+fig.basemap(region=region, projection=projection, frame=frame)
+
+fig.meca(
+    spec=aki_dict_single,
+    scale="1c",
+    longitude=-2,
+    latitude=0,
+    nodal="0/1p,black,solid",
+)
+
+fig.meca(
+    spec=aki_dict_single,
+    scale="1c",
+    longitude=2,
+    latitude=0,
+    compressionfill="lightorange",
+    outline="0.5p,black,solid",
+)
+fig.meca(
+    spec=aki_dict_single,
+    scale="1c",
+    longitude=2,
+    latitude=0,
+    nodal="1/1p,darkorange,solid",
+)
+fig.meca(
+    spec=aki_dict_single,
+    scale="1c",
+    longitude=2,
+    latitude=0,
+    compressionfill="white@100",
+    extensionfill="white@100",
+    pen="1p,gray30,solid",
+)
+fig.show()
+
+
+# %%
+# Adding offset from event location
+# ---------------------------------
+#
+# Specify the optional parameters ``plot_longitude`` and ``plot_latitude``.
+# Additionally the parameter ``offset`` has to be set. Besides just drawing a line
+# between the beachball and the event location, a small circle can be plotted at the
+# event location by appending **+s** and the descired circle diameter. The connecting
+# line as well as the outline of the circle are plotted with the  setting of pen, or
+# can be adjusted separately. The fill of the small circle corresponds to the fill
+# of the compressive quadrantes.
+
+fig = pygmt.Figure()
+fig.basemap(region=region, projection=projection, frame=frame)
+
+fig.meca(
+    spec=aki_dict_single,
+    scale="1c",
+    longitude=-1,
+    latitude=0,
+    plot_longitude=-3,
+    plot_latitude=2,
+    offset=True,
+)
+
+fig.meca(
+    spec=aki_dict_single,
+    scale="1c",
+    longitude=3,
+    latitude=0,
+    plot_longitude=1,
+    plot_latitude=2,
+    offset="+p1p,darkorange+s0.25c",
+    compressionfill="lightorange",
+)
+
+fig.show()
```

Comment:
Looking at the preview at https://pygmt-dev--2550.org.readthedocs.build/en/2550/tutorials/advanced/focal_mechanisms.html#adding-offset-from-event-location, there seem to be a new issue with the offset when using GMT 6.5.0 and PyGMT dev / v0.14.2. I can reproduce this issue locally (under Windows), but on Linux using GMT 6.5.0 and PyGMT v0.14.0. is works as expected:

| GMT dev / 6.5.0 and PyGMT dev / v0.14.2 | GMT 6.5.0 and PyGMT v0.14.0 |
| --- | --- |
| ![meca_offset_gmt650_pygmt0142](https://github.com/user-attachments/assets/81e7f445-3714-460c-802a-0588b053f138) | ![meca_offset_gmt650_pygmt0140](https://github.com/user-attachments/assets/3d57873a-b47b-4d67-b940-d1b174837d9a) |

Reply:
The preview shows beachballs with offsetting. Am I misunderstanding here?

Reply:
The image in the left column of the table is the image currently shown in the preview of this PR. There the beachballs are plotted at the positions given by `longitude` and `latitude`. But actually, the beachballs should be plotted at the positions given by `plot_longitude` and `plot_latitude`, and there should be a line and a small circle highlighting the even location. This is shown in the image in the right column of the table. For me, it works with v0.14.0, but not with v0.14.2.

Reply:
> The image in the left column of the table is the image currently shown in the preview of this PR.

Hmm, I still see the right figure in the preview docs. I also tried it locally with PyGMT dev, and got the right one.

Reply:
Hm. Somehow wired, when using another browser, the preview looks correct. Anyways, seems like the issue is on my side, so this should be OK 🙂.